### DISCUSSION TITLE
File List Greyed or Blacked Out, (depending on theme), when Launching from Icon (launcher)

### DISCUSSION THREAD

**Discussion by levone1** [jarun/nnn#1626] [Q&A]
Probably easier than I realize, but I can't figure it out..  When I launch nnn from terminal, opens and looks fine, whatever the parameters.  I usually use Kitty, but tried several others also.  When I make a .desktop file, and nnn shows up in application launcher, it launches and works fine, but all of the words in the file list are blocked out with a color overlay.  Color changes depending on the theme of the terminal, etc.  I have tried several variants of desktop file, and changing bashrc, zshrc, export NNN..., etc., but always the same.  Also is the same across users, and whether or not my desktop file is in ~/.local/share, or in /usr...  I have also tried changing default terminal program, and SHELL and TERM variables.  Also tried changing command in desktop file to `kitty -e ...` Any ideas what's going on?

Attachments show the desktop file, (example - I've tried several mods), and results from opening with desktop file, and opening with terminal command, (same command as in desktop file).

Linux Mint 21
nnn version 4.7 built from source, and also the same with 4.3 from APT
same on kitty or mate-terminal or konsole, etc

Thanks

![launch_with_launcher](https://user-images.githubusercontent.com/43003405/231039435-3a9d48aa-9e78-4971-8762-cff2146595ac.png)
![launch_with_command](https://user-images.githubusercontent.com/43003405/231039438-7eee232c-0052-485c-939a-ab3dd9ed1e88.png)
![command](https://user-images.githubusercontent.com/43003405/231039440-4883bf19-f153-4aa9-9001-0521c864944a.png)
![desktop_file](https://user-images.githubusercontent.com/43003405/231039441-4f386f09-a93d-4241-bb2f-9c8cfc738a8d.png)

**Comment by levone1** [comment]
Well, never figured it out, but I did solve the issue by eliminating all traces of nnn and starting over, so all better now...  Still curious, if anyone has ideas.

**Comment by jarun** [comment]
Seems like a local env issue to me. Maybe some color setting?
